### ISSUE TITLE
CalAmp Fusion Tracking

### ISSUE THREAD

**Issue by pmiller10004** [tananaev/traccar#459]
I have a CalAmp Fusion device and was trying to set this up locally in LAB.  CalAmp I believe has their own implementation of opengts however, I wanted to get this working with traccar.  Specifically i think i need to add different udp ports as stated in their manual but need some quick advice on how to do this?
Their Docs:  http://www.geotelematic.com/docs/DCServer_CalAmp.pdf

thanks

**Comment by tananaev** [comment]
CalAmp protocol is now supported by Traccar. If you are interested I can provide a build to try it.

**Comment by nwohaibi** [comment]
I have several Calamp OBD devices, I could use a build to test the protocol implementation? UDP right?

**Comment by tananaev** [comment]
Yes, UDP protocol. Here is a JAR file:
https://www.dropbox.com/s/cgmlansv32j3ii0/tracker-server.jar?dl=0

You need to add following lines to the config file:
<pre>    &lt;entry key='calamp.enable'&gt;true&lt;/entry&gt;
    &lt;entry key='calamp.port'&gt;5082&lt;/entry&gt;</pre>

**Comment by nwohaibi** [comment]
I think I am facing an issue with the format sent by Calamp LMU device:
```
2015-04-12 17:51:50 DEBUG: [B8881EC1: 5000 <- x.x.x.x] - HEX: 830545321041830101010300010000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-12 17:51:50  INFO: Protocol gps103 no match
2015-04-12 17:51:50  INFO: Protocol gps103 no match
2015-04-12 17:51:50  INFO: Protocol tk103 no match
2015-04-12 17:51:50  INFO: Protocol tk103 no match
2015-04-12 17:51:50  INFO: Protocol gl100 no match
2015-04-12 17:51:50  INFO: Protocol gl200 no match
2015-04-12 17:51:50  INFO: Protocol t55 no match
2015-04-12 17:51:50  INFO: Protocol xexun no match
2015-04-12 17:51:50  INFO: Protocol totem no match
2015-04-12 17:51:50  INFO: Protocol enfora no match
2015-04-12 17:51:50  INFO: Protocol meiligao no match
2015-04-12 17:51:50  INFO: Protocol maxon no match
2015-04-12 17:51:50  INFO: Protocol suntech no match
2015-04-12 17:51:50  INFO: Protocol progress no match
2015-04-12 17:51:50  INFO: Protocol h02 no match
2015-04-12 17:51:50  INFO: Protocol jt600 error
2015-04-12 17:51:50  INFO: Protocol ev603 no match
2015-04-12 17:51:50  INFO: Protocol v680 no match
2015-04-12 17:51:50  INFO: Protocol pt502 no match
2015-04-12 17:51:50  INFO: Protocol tr20 no match
2015-04-12 17:51:50  INFO: Protocol navis no match
2015-04-12 17:51:50  INFO: Protocol meitrack error
2015-04-12 17:51:50  INFO: Protocol skypatrol no match
2015-04-12 17:51:50  INFO: Protocol gt02 no match
2015-04-12 17:51:50  INFO: Protocol gt06 no match
2015-04-12 17:51:50  INFO: Protocol megastek no match
2015-04-12 17:51:50  INFO: Protocol navigil no match
2015-04-12 17:51:50  INFO: Protocol gpsgate no match
2015-04-12 17:51:50  INFO: Protocol teltonika no match
2015-04-12 17:51:50  INFO: Protocol mta6 error
2015-04-12 17:51:50  INFO: Protocol mta6can error
2015-04-12 17:51:50  INFO: Protocol tlt2h no match
2015-04-12 17:51:50  INFO: Protocol syrus no match
2015-04-12 17:51:50  INFO: Protocol wondex no match
2015-04-12 17:51:50  INFO: Protocol cellocator no match
2015-04-12 17:51:50  INFO: Protocol galileo no match
2015-04-12 17:51:50  INFO: Protocol ywt no match
2015-04-12 17:51:50  INFO: Protocol tk102 no match
2015-04-12 17:51:50  INFO: Protocol intellitrac no match
2015-04-12 17:51:50  WARN: Unknown device - 2A�38b
2015-04-12 17:51:50  INFO: Protocol xt7 no match
2015-04-12 17:51:50  INFO: Protocol wialon no match
2015-04-12 17:51:50  INFO: Protocol carscop no match
2015-04-12 17:51:50  INFO: Protocol apel no match
2015-04-12 17:51:50  INFO: Protocol manpower no match
2015-04-12 17:51:50  INFO: Protocol globalsat no match
2015-04-12 17:51:50  INFO: Protocol atrack no match
2015-04-12 17:51:50  WARN: Unknown device - 72342367566299187
2015-04-12 17:51:50  INFO: Protocol atrack no match
2015-04-12 17:51:50  INFO: Protocol pt3000 no match
2015-04-12 17:51:50  INFO: Protocol ruptela no match
2015-04-12 17:51:50  INFO: Protocol topflytech no match
2015-04-12 17:51:50  INFO: Protocol laipac no match
2015-04-12 17:51:50  INFO: Protocol aplicom no match
2015-04-12 17:51:50  INFO: Protocol gotop no match
2015-04-12 17:51:50  INFO: Protocol sanav no match
2015-04-12 17:51:50  INFO: Protocol gator no match
2015-04-12 17:51:50  INFO: Protocol noran no match
2015-04-12 17:51:50  WARN: Unknown device - 9-2-73718-1-63399-3-1
2015-04-12 17:51:50  INFO: Protocol m2m no match
2015-04-12 17:51:50  INFO: Protocol osmand error
2015-04-12 17:51:50  INFO: Protocol easytrack no match
2015-04-12 17:51:50  INFO: Protocol taip no match
2015-04-12 17:51:50  INFO: Protocol khd no match
2015-04-12 17:51:50  INFO: Protocol piligrim error
2015-04-12 17:51:50  INFO: Protocol stl060 no match
2015-04-12 17:51:50  INFO: Protocol cartrack no match
2015-04-12 17:51:50  INFO: Protocol minifinder no match
2015-04-12 17:51:50  INFO: Protocol haicom no match
2015-04-12 17:51:50  INFO: Protocol eelink no match
2015-04-12 17:51:50  INFO: Protocol box no match
2015-04-12 17:51:50  INFO: Protocol freedom no match
2015-04-12 17:51:50  INFO: Protocol telik no match
2015-04-12 17:51:50  INFO: Protocol trackbox no match
2015-04-12 17:51:50  INFO: Protocol visiontek no match
2015-04-12 17:51:50  INFO: Protocol orion no match
2015-04-12 17:51:50  INFO: Protocol riti no match
2015-04-12 17:51:50  INFO: Protocol ulbotech no match
2015-04-12 17:51:50  INFO: Protocol tramigo no match
2015-04-12 17:51:50  INFO: Protocol tr900 no match
2015-04-12 17:51:50  INFO: Protocol ardi01 no match
2015-04-12 17:51:50  INFO: Protocol calamp no match
2015-04-12 17:55:50 DEBUG: [B8881EC1: 5000 <- x.x.x.x] - HEX: 830545321041830101010300010000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-12 17:55:50  INFO: Protocol gps103 no match
2015-04-12 17:55:50  INFO: Protocol gps103 no match
2015-04-12 17:55:50  INFO: Protocol tk103 no match
2015-04-12 17:55:50  INFO: Protocol tk103 no match
2015-04-12 17:55:50  INFO: Protocol gl100 no match
2015-04-12 17:55:50  INFO: Protocol gl200 no match
2015-04-12 17:55:50  INFO: Protocol t55 no match
2015-04-12 17:55:50  INFO: Protocol xexun no match
2015-04-12 17:55:50  INFO: Protocol totem no match
2015-04-12 17:55:50  INFO: Protocol enfora no match
2015-04-12 17:55:50  INFO: Protocol meiligao no match
2015-04-12 17:55:50  INFO: Protocol maxon no match
2015-04-12 17:55:50  INFO: Protocol suntech no match
2015-04-12 17:55:50  INFO: Protocol progress no match
2015-04-12 17:55:50  INFO: Protocol h02 no match
2015-04-12 17:55:50  INFO: Protocol jt600 error
2015-04-12 17:55:50  INFO: Protocol ev603 no match
2015-04-12 17:55:50  INFO: Protocol v680 no match
2015-04-12 17:55:50  INFO: Protocol pt502 no match
2015-04-12 17:55:50  INFO: Protocol tr20 no match
2015-04-12 17:55:50  INFO: Protocol navis no match
2015-04-12 17:55:50  INFO: Protocol meitrack error
2015-04-12 17:55:50  INFO: Protocol skypatrol no match
2015-04-12 17:55:50  INFO: Protocol gt02 no match
2015-04-12 17:55:50  INFO: Protocol gt06 no match
2015-04-12 17:55:50  INFO: Protocol megastek no match
2015-04-12 17:55:50  INFO: Protocol navigil no match
2015-04-12 17:55:50  INFO: Protocol gpsgate no match
2015-04-12 17:55:50  INFO: Protocol teltonika no match
2015-04-12 17:55:50  INFO: Protocol mta6 error
2015-04-12 17:55:50  INFO: Protocol mta6can error
2015-04-12 17:55:50  INFO: Protocol tlt2h no match
2015-04-12 17:55:50  INFO: Protocol syrus no match
2015-04-12 17:55:50  INFO: Protocol wondex no match
2015-04-12 17:55:50  INFO: Protocol cellocator no match
2015-04-12 17:55:50  INFO: Protocol galileo no match
2015-04-12 17:55:50  INFO: Protocol ywt no match
2015-04-12 17:55:50  INFO: Protocol tk102 no match
2015-04-12 17:55:50  INFO: Protocol intellitrac no match
2015-04-12 17:55:50  WARN: Unknown device - 2A�38b
2015-04-12 17:55:50  INFO: Protocol xt7 no match
2015-04-12 17:55:50  INFO: Protocol wialon no match
2015-04-12 17:55:50  INFO: Protocol carscop no match
2015-04-12 17:55:50  INFO: Protocol apel no match
2015-04-12 17:55:50  INFO: Protocol manpower no match
2015-04-12 17:55:50  INFO: Protocol globalsat no match
2015-04-12 17:55:50  INFO: Protocol atrack no match
2015-04-12 17:55:50  WARN: Unknown device - 72342367566299187
2015-04-12 17:55:50  INFO: Protocol atrack no match
2015-04-12 17:55:50  INFO: Protocol pt3000 no match
2015-04-12 17:55:50  INFO: Protocol ruptela no match
2015-04-12 17:55:50  INFO: Protocol topflytech no match
2015-04-12 17:55:50  INFO: Protocol laipac no match
2015-04-12 17:55:50  INFO: Protocol aplicom no match
2015-04-12 17:55:50  INFO: Protocol gotop no match
2015-04-12 17:55:50  INFO: Protocol sanav no match
2015-04-12 17:55:50  INFO: Protocol gator no match
2015-04-12 17:55:50  INFO: Protocol noran no match
2015-04-12 17:55:50  WARN: Unknown device - 9-2-73718-1-63399-3-1
2015-04-12 17:55:50  INFO: Protocol m2m no match
2015-04-12 17:55:50  INFO: Protocol osmand error
2015-04-12 17:55:50  INFO: Protocol easytrack no match
2015-04-12 17:55:50  INFO: Protocol taip no match
2015-04-12 17:55:50  INFO: Protocol khd no match
2015-04-12 17:55:50  INFO: Protocol piligrim error
2015-04-12 17:55:50  INFO: Protocol stl060 no match
2015-04-12 17:55:50  INFO: Protocol cartrack no match
2015-04-12 17:55:50  INFO: Protocol minifinder no match
2015-04-12 17:55:50  INFO: Protocol haicom no match
2015-04-12 17:55:50  INFO: Protocol eelink no match
2015-04-12 17:55:50  INFO: Protocol box no match
2015-04-12 17:55:50  INFO: Protocol freedom no match
2015-04-12 17:55:50  INFO: Protocol telik no match
2015-04-12 17:55:50  INFO: Protocol trackbox no match
2015-04-12 17:55:50  INFO: Protocol visiontek no match
2015-04-12 17:55:50  INFO: Protocol orion no match
2015-04-12 17:55:50  INFO: Protocol riti no match
2015-04-12 17:55:50  INFO: Protocol ulbotech no match
2015-04-12 17:55:50  INFO: Protocol tramigo no match
2015-04-12 17:55:50  INFO: Protocol tr900 no match
2015-04-12 17:55:50  INFO: Protocol ardi01 no match
2015-04-12 17:55:50  INFO: Protocol calamp no match
2015-04-12 17:59:50 DEBUG: [D7669900: 5082 <- x.x.x.x] - HEX: 830545321041830101010300010000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-12 17:59:50  WARN: Unknown device - 4532104183
2015-04-12 17:59:50  WARN: Unknown device
2015-04-12 18:03:50 DEBUG: [D7669900: 5082 <- x.x.x.x] - HEX: 830545321041830101010300010000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-12 18:03:50  WARN: Unknown device - 4532104183
2015-04-12 18:03:50  WARN: Unknown device
2015-04-12 18:11:28 DEBUG: [9F1FA912: 5004 <- x.x.x.] - HEX: 2b524553503a47545354542c3146303130382c383634...............
```

**Comment by nwohaibi** [comment]
The log shows both ports 5000 and 5082 do not recognize Calamp OBD device.
While port 5004 confirms that the server is working with other devices ..

**Comment by nwohaibi** [comment]
@tananaev 
```
2015-04-13 10:02:08 DEBUG: [75CD7525: 5004 <- x.x.x.x] - HEX: 2b524553503a47544652492c3146303130382c3836343235313032303039353936382c2c2c2c31302c312c312c302e302c323.....
2015-04-13 10:02:08  INFO: device: 15, time: Mon Apr 13 10:02:03 UTC 2015, lat: 26.xxx, lon: 50.xxx
...
2015-04-13 11:15:50 DEBUG: [AB0E4B64: 5004 <- x.x.x.x] - HEX: 2b41434b3a47544f534d2c3146303130372c3836343235313032303034383831392c2c303034322c32303131303130313030303230352c4645324524
2015-04-13 11:15:50  INFO: Closing connection by disconnect
2015-04-13 11:15:51 DEBUG: [C74EC97A: 5004 <- x.x.x.x] - HEX: 2b524553503a47544652492c3146303130372c3836343235313032303034383831392c2c2c2c31302c312c302c2c2c2c302c302c2c303432302c303030342c353635352c344646382c30302c302e302c2c2c2c302c3432303030302c2c2c2c32303131303130313030303230352c4645324624
2015-04-13 11:17:25 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 830545321041830101010300010000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-13 11:17:25  WARN: Unknown device - 4532104183
2015-04-13 11:17:25  WARN: Unknown device
2015-04-13 11:21:25 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 830545321041830101010300010000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-13 11:21:25  WARN: Unknown device - 4532104183
2015-04-13 11:21:25  WARN: Unknown device
2015-04-13 11:22:24 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 830545321041830101010300010000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-13 11:22:24  WARN: Unknown device - 4532104183
2015-04-13 11:22:24  WARN: Unknown device
2015-04-13 11:22:37 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 830545321041830101010300010000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-13 11:22:37  WARN: Unknown device - 4532104183
2015-04-13 11:22:37  WARN: Unknown device
2015-04-13 11:26:53 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 830545321041830101010300010000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-13 11:26:53  WARN: Unknown device - 4532104183
2015-04-13 11:26:53  WARN: Unknown device
```

**Comment by nwohaibi** [comment]
I have just added 4532104183 as an IMEI into Traccar, however this is not the device real IMEI.
Do you recommend any configuration in CalAmp LMU manager?
```
2015-04-13 11:38:53  WARN: Unknown device - 4532104183
2015-04-13 11:38:53  WARN: Unknown device
2015-04-13 12:01:47 DEBUG: [033671A8: 5004 <- x.x.x.x] - HEX: 2b524553503a47544652492c3146303130382c38363432353130323030393...
2015-04-13 12:01:47  INFO: device: 15, time: Mon Apr 13 12:01:42 UTC 2015, lat: 26.xxx, lon: 50.xxx
2015-04-13 12:01:48  INFO: Closing connection by disconnect
2015-04-13 12:03:41 DEBUG: [D7669900: 5082 <- xx.xx.xx.xx] - HEX: 66726f6d2075647020746573746572
2015-04-13 12:03:41  WARN: Unknown device
2015-04-13 12:22:28 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 830545321041830101010300010000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-13 12:22:28 DEBUG: [D7669900: 5082 -> y.y.y.y] - HEX: 02010001030000000000
2015-04-13 12:22:29 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 830545321041830101010300020000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-13 12:22:29 DEBUG: [D7669900: 5082 -> y.y.y.y] - HEX: 02010002030000000000
2015-04-13 12:22:29 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 830545321041830101010300040000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-13 12:22:29 DEBUG: [D7669900: 5082 -> y.y.y.y] - HEX: 02010004030000000000
2015-04-13 12:22:30 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 8305453210418301010105000555269733552697340ebfe5611be26c700000ebc100000002003107200004ffcd0f0a0000006f00834150503a4c4d552c3139352056332e386220284a756e20313920323031342031363a32393a313629200d0a5049433a53544d20532f4e2034353332313034313833200d0a4750533a55424c4f582d3030303730303030202d312e30302028353938343229200d0a526164696f3a752d626c6f7820534152412d473335302d30382e3439
2015-04-13 12:22:30 DEBUG: [D7669900: 5082 -> y.y.y.y] - HEX: 02010005050000000000
2015-04-13 12:22:31 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 830545321041830101010300060000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-13 12:22:31 DEBUG: [D7669900: 5082 -> y.y.y.y] - HEX: 02010006030000000000
2015-04-13 12:22:32 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 830545321041830101010300070000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-13 12:22:32 DEBUG: [D7669900: 5082 -> y.y.y.y] - HEX: 02010007030000000000
2015-04-13 12:22:32 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 830545321041830101010300080000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-13 12:22:32 DEBUG: [D7669900: 5082 -> y.y.y.y] - HEX: 02010008030000000000
2015-04-13 12:22:33 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 830545321041830101010300090000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-13 12:22:33 DEBUG: [D7669900: 5082 -> y.y.y.y] - HEX: 02010009030000000000
2015-04-13 12:22:34 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 8305453210418301010103000b0000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-13 12:22:34 DEBUG: [D7669900: 5082 -> y.y.y.y] - HEX: 0201000b030000000000
2015-04-13 12:22:38 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 8305453210418301010105000c5526a6255526a6250ebfe1da1be26e530000ebc200000005003407200004ffcd0f0c0000006f00834150503a4c4d552c3139352056332e386220284a756e20313920323031342031363a32393a313629200d0a5049433a53544d20532f4e2034353332313034313833200d0a4750533a55424c4f582d3030303730303030202d312e30302028353938343229200d0a526164696f3a752d626c6f7820534152412d473335302d30382e3439
2015-04-13 12:22:38 DEBUG: [D7669900: 5082 -> y.y.y.y] - HEX: 0201000c050000000000
2015-04-13 12:22:38 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 8305453210418301010103000d0000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-13 12:22:38 DEBUG: [D7669900: 5082 -> y.y.y.y] - HEX: 0201000d030000000000
2015-04-13 12:22:39 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 8305453210418301010103000e0000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-13 12:22:39 DEBUG: [D7669900: 5082 -> y.y.y.y] - HEX: 0201000e030000000000
2015-04-13 12:22:40 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 830545321041830101010300100000333862000023c301000000004532104183ffffff353816051610691f420040163953294fffffffffffffffff8996604211639032949f4f54413a317c303b302c317c343b302c34004f5441535441543a302c302c302c302c302c222200564255533a342c322e302e302c343533323130343138332c5630312e30332e30312e34302c5630312e30332e30312e33312c2c0056494e2d494e464f3a56494e3d31464d5a5537324539355a4137303032362c4445562d5245474e3d55532c535256522d5245474e3d555300
2015-04-13 12:22:40 DEBUG: [D7669900: 5082 -> y.y.y.y] - HEX: 02010010030000000000
2015-04-13 12:22:40 DEBUG: [D7669900: 5082 <- y.y.y.y] - HEX: 8305453210418301010105001155297a4f55297a4f0fac73d31de4c40200000d2e0000000b00af0a200004ffc70f080000006f00834150503a4c4d552c3139352056332e386220284a756e20313920323031342031363a32393a313629200d0a5049433a53544d20532f4e2034353332313034313833200d0a4750533a55424c4f582d3030303730303030202d312e30302028353938343229200d0a526164696f3a752d626c6f7820534152412d473335302d30382e3439
2015-04-13 12:22:40 DEBUG: [D7669900: 5082 -> y.y.y.y] - HEX: 02010011050000000000

```

**Comment by tananaev** [comment]
Your device sends only ID Report Messages (Message Type 3). Also, looks like you haven't registered it in the database:
<pre>Unknown device - 4532104183</pre>

**Comment by scmachines** [comment]
I'm not sure if I'm posting in the correct place, but I have a related issue. I'm trying to get some Calamp GPS devices to talk to my new install of OpenGTS on Ubuntu 14.04. I had no Calamp-specific files to work with, so I'm running the server as "template.jar" with the Calamp port and other settings pasted into the dcservers.xml file. When I start the DCS server, I get a pid and bin/psjava shows the server is running - but sudo netstat -plnt does not show the listening port. I have also tried starting servers like aspicore (for test purposes) that a jar file exists for - but no listening port shows up. These are the instructions I've cobbled together in my implementation:http://linoxide.com/security/setup-opengts-gps-tracking-ubuntu-centos/ and http://www.geotelematic.com/docs/DCServer_CalAmp.pdf . We are just in a testing phase with two GPS devices inside of vehicles and configured to send their data to port 20500. The vendor is not able to be much help with any software, and I am not a software developer by profession. Is there something I'm missing that is preventing the port from listening? Thanks...
